DDDEastAnglia.Tests/Services/Messengers/Email/EmailMessengerFacts.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net.Mail;
+using DDDEastAnglia.Helpers;
+using DDDEastAnglia.Models;
+using DDDEastAnglia.Services.Messenger.Email;
+using NSubstitute;
+using NUnit.Framework;
+using MailMessage = DDDEastAnglia.Services.Messenger.Email.MailMessage;
+
+namespace DDDEastAnglia.Tests.Services.Messengers.Email
+{
+    [TestFixture]
+    public class EmailMessengerFacts
+    {
+        [TestFixture]
+        public class Notify_Should
+        {
+            [Test]
+            public void Prepare_The_Mail_For_Delivery()
+            {
+                var postman = Substitute.For<IPostman>();
+                var template = Substitute.For<IMailTemplate>();
+                template.RenderBody().Returns("Message body");
+                template.RenderSubjectLine().Returns("Message subject");
+
+                var messenger = new EmailMessenger(postman, template);
+                messenger.Notify(new UserProfile {EmailAddress = "speaker@dddeastanglia.com"});
+
+                var message = new MailMessage
+                {
+                    From = new MailAddress("admin@dddeastanglia.com", "DDD East Anglia"),
```

Comment:
Assuming we care about that (which I think we do), then yes. Otherwise we can remove it from the equality calculation.